### PR TITLE
Update version examples and add branding documentation

### PR DESCRIPTION
**Pull request by SahAssar**

### CODE REVIEW

**Comment by SahAssar** [comment]
Perhaps branding should move to another section if it continues to grow?

File: source/howto/performance-tips.md [hunk 1]
```diff
@@ -72,18 +72,31 @@ have to resize images on each request, but there _is_ a certain overhead in this
...
   name: Bolt
   path: /admin
   provided_by: [ supercool@example.org, "Supercool Webdesign Co." ]
+  news_source: http://news.example.org
+  news_variable: news
+  favicon: /files/icons/favicon.ico
+  apple-touch-icon: /files/icons/apple-touch-icon.png
+  apple-touch-icon-72x72: /files/icons/apple-touch-icon-72x72.png
+  apple-touch-icon-114x114: /files/icons/apple-touch-icon-114x114.png
+  apple-touch-icon-144x144: /files/icons/apple-touch-icon-144x144.png
```
Comment (bobdenotter) [inline source/howto/performance-tips.md]: Can we do this? When did this happen? ^_^

**Comment by SahAssar** [comment]
Haha, Well, the news stuff was added in a PR by me not long ago, and the favicon/apple-touch-icons are here: https://github.com/bolt/bolt/blob/master/app/view/twig/_base/_page.twig#L71-L75 and were added by schwarzmedia on Dec 29, 2013, see here: https://github.com/bolt/bolt/commit/b33179f4ba2d54c7af560a44b0748bf3c65357a2

So it seems like it has been possible but undocumented for quite a while. I found it by just digging through the code. I just assumed you guys knew about it :smiley:

**Comment by bobdenotter** [comment]
I forgot about that! Well, good to have it documented, then! :+1:

**Pull request merged**